defaults/main.yml

```diff
@@ -2,24 +2,26 @@
 ---
 network_connections: []
 
-network_allow_restart: no
+network_allow_restart: false
+
+# List of RedHat family distributions supported by the role
```

Comment:
It seems better to avoid "supported" here, since only RedHat and CentOS are supported. The others are recognized as RHEL clones without investing in supporting them.

Reply:
> It seems better to avoid "supported" here, since only RedHat and CentOS are supported. The others are recognized as RHEL clones without investing in supporting them.

ok - what wording would you suggest?